src/mip/ensure_integer_bounds.jl

```diff
@@ -0,0 +1,34 @@
+function ceil(val::T, ϵ_int::T)::T where{T}
```

Comment:
The punning on ``Base.ceil`` is a bit confusing. Could you change the naming here to something like ``_ceil`` or ``approx_ceil``? (and similarly below for ``floor``).